src/sql/sqlops.proposed.d.ts

```diff
@@ -595,6 +606,7 @@ declare module 'sqlops' {
 		html: string;
 		message: any;
 		onMessage: vscode.Event<any>;
+		readonly options: vscode.WebviewOptions;
```

Comment:
I'm not sure the context of these changes, but readonly in TS only means the objects can't be reassigned, the object is still mutable.

Reply:
As mentioned offline, making this readonly since the user ideally should only set on initial creation using `withProperties`. 

Anthony - Currently the only way to get the UI to update is to change one of the properties directly rather than mutate any inner field of that property. This conforms to that limitation (though it's a shame that this is the current behavior).